eox_tenant/test/test_pipeline.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+"""
+Tests for the pipeline module used in multi-tenant third party auth.
+"""
+from django.test import TestCase
+from mock import MagicMock
+
+from eox_tenant.pipeline import safer_associate_by_email, EoxTenantAuthException
+
+
+class AssociationByEmailTest(TestCase):
+    """
+    Test the custom association backend.
+    """
+    def setUp(self):
+        self.backend_mock = MagicMock()
+        self.user_mock = MagicMock()
+        self.backend_mock.strategy.storage.user.get_users_by_email.return_value = [self.user_mock]
+
+    def test_regular_user_works(self):
+        """
+        A non admin user will be assigned and returned.
+        """
+        self.user_mock.is_staff = False
+        self.user_mock.is_superuser = False
+        result = safer_associate_by_email(self.backend_mock, {'email': 'fake@example.com'})
+        self.assertEquals(self.user_mock, result['user'])
```

Comment:
I think `assertEqual` should work as well. This is to fix the quality issue the code currently has